client/src/components/pages/provider/ProviderActiveAlertsPage.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { getAlerts } from 'actions';
+import { AlertTable } from 'table';
+
+class ProviderActiveAlertsPage extends React.Component {
+
+    componentWillMount() {
+        this.props.getAlerts();
+    }
+
+    render() {
+        let { alerts } = this.props;
+        const description = (
+            'Here you can see alerts you have responded ' +
+            'to and any alerts within the past 2 days ' +
+            'that you have not responded to.'
+        );
+        return (
+            <AlertTable 
+              alerts={alerts}
+              role='provider'
+              title='Your Alerts'
```

Comment:
No it's just un-responded alerts. I'll go with just alerts. There will probably be another page that is like "Responded Alerts" or "Alert History" that shows their alert history.